app/src/scenes/ConsoFollowUp/Diagram.js

```diff
@@ -23,7 +23,9 @@ const maxDosesOnScreen = 999;
 
 const computeBarsHeight = (highestDosesInPeriod, highestAcceptableDosesPerDay) => {
   const barHighestHeightPossible = screenHeight * 0.2;
-
+  if (highestDosesInPeriod < 2) {
+    highestDosesInPeriod = 2;
+  }
   return {
     barMaxHeight: barHighestHeightPossible,
     barMaxAcceptableDoseHeight: (highestAcceptableDosesPerDay / highestDosesInPeriod) * barHighestHeightPossible,
```

Comment:
C'esst toujours un peu bizarre de set un paramètre. j'aurai mis direct en ternaire (ou dans une var temporaire) mais si tu n'as pas de pb tu peux livrer comme tu as fait

```suggestion
    barMaxAcceptableDoseHeight: (highestAcceptableDosesPerDay / (highestDosesInPeriod < 2 ? 2 : highestDosesInPeriod)) * barHighestHeightPossible,
```